src/ensembl/src/content/app/browser/track-panel/TrackPanel.test.tsx

```diff
@@ -41,19 +41,32 @@ describe('<TrackPanel />', () => {
     toggleTrackPanel: jest.fn()
   };
 
-  const mountBrowserImageComponent = (props?: Partial<TrackPanelProps>) =>
+  const mountTrackPanel = (props?: Partial<TrackPanelProps>) =>
     mount(<TrackPanel {...defaultProps} {...props} />);
 
   describe('rendering', () => {
     test('renders track panel when active genome is present', () => {
-      const wrapper = mountBrowserImageComponent({
+      const wrapper = mountTrackPanel({
         activeGenomeId: faker.lorem.words()
       });
       expect(wrapper.html()).not.toBe(null);
     });
 
+    test('shows track panel only if screen is or bigger than big desktop', () => {
```

Comment:
"if screen is or bigger than big desktop" -> "if the screen width is big desktop or larger" 😃